packages/mux-player/src/video-api.ts

```diff
@@ -250,15 +250,7 @@ class VideoApiElement extends HTMLElement {
   }
 
   set autoplay(val) {
-    if (val) {
-      this.setAttribute(
-        AllowedVideoAttributes.AUTOPLAY,
-        typeof val === "string" ? val : ""
-      );
-    } else {
-      // Remove boolean attribute if false, 0, '', null, undefined.
-      this.removeAttribute(AllowedVideoAttributes.AUTOPLAY);
-    }
+    this.setAttribute(AllowedVideoAttributes.AUTOPLAY, `${val}`);
```

Comment:
blocking: `autoplay` can be either a string or a boolean… I think this isn't quite what we want here.

Reply:
a boolean is an empty string and it's up to the user to set that I think. it's more inline with the native HTML elements

Reply:
as long when in mux-player-react when the prop is set to `null` this attribute will end up getting removed.

Reply:
if I have a `<video/>` tag and use its setter for `autoplay`, I can/would use `videoEl.autoplay = true` or `videoEl.autoplay = false`, so I'd assume that would be more in line with native HTML elements when it comes to setters, right?

Reply:
hmm yes, autoplay = false would still enable it. I guess this is an edge case with different types could be used

closing then